tests-integration/tests/http.rs

```diff
@@ -338,51 +337,6 @@ pub async fn test_metrics_api(store_type: StorageType) {
     guard.remove_all().await;
 }
 
-pub async fn test_catalog_metrics(store_type: StorageType) {
-    common_telemetry::init_default_ut_logging();
-    common_telemetry::init_default_metrics_recorder();
-
-    let (app, mut guard) = setup_test_http_app(store_type, "metrics_api").await;
```

Comment:
I think we should add back this test, not for the catalog metrics but to ensure the `/metrics` API is working.

We can change the assertion.